gradle/libs.versions.toml

```diff
@@ -37,7 +37,7 @@ guava = { module = "com.google.guava:guava", version = "33.4.0-jre" }
 
 kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
 kotlinpoet-ksp = { module = "com.squareup:kotlinpoet-ksp", version = "2.0.0" }
-kotlin-compile-testing-ksp = { module = "com.github.tschuchortdev:kotlin-compile-testing-ksp", version = "1.6.0" }
```

Comment:
this is needed as it supports kotlin 2.0. The previous version used kotlin 1.9.x